src/components/internals/create-select-styles.js

```diff
@@ -337,7 +337,7 @@ const multiValueRemoveStyles = (props, theme) => base => {
     } 0`,
     borderStyle: 'solid',
     borderWidth: '1px',
-
+    pointerEvents: state.isDisabled ? 'none' : base.pointerEvents,
```

Comment:
another option would be in TagRemove removing the onClick from the spread props 🤔

Reply:
No opinion on it. I think pointerEvents should be `none` anyway?

Reply:
True, it's not one or the other, it's one or both 😄

Reply:
If it makes it easier to test I'd `null` the `onChange` too.